docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -231,29 +231,29 @@ V&V Checks:
     -    
   * - 3
     - % of each facility type have cesarian section capabilities
-    - Public/governmental health facility (62%), private and/or NGO health facility (73%), at home  (0%), other (50%) 
+    - Public/governmental health facility (62%), private for-profit health facility (60%), at home  (0%), private not-for-profit health facility (86%), other (10%)  
     - EmONC (Ethiopia; saved at J:\Project\simulation_science\optimization_model_data\EmONC 2016_Master Dataset_Final\ETH_EmONC Assessment 2016_Final Report Jan11 2018.pdf) 
     - We want these to be location specific, please code accordingly. These are placeholder values for now (extracted from the EmONC Final Report, link in 'Source' column; the 'other' value is made-up), hopefully we will be able to find similar data available for Pakistan and Nigeria.   
   * - 4
-    - XX relative risk on incidence of hemorrhage and obstructed labor 
+    - XX relative risk of c-section on incidence of hemorrhage and obstructed labor 
     - 
     - 
     - Outstanding items: how does c-section need overlap with hemorrhage/OL, what is the RR, how will we implement this with overlaps in total MD impact of facility type 
   * - 5
-    - XX% of pregnancy receive in each delivery facility type
-    - 
-    - 
-    - Confirm understanding that all pregnancies can/should receive this
+    - % of pregnancy receive azithromycin in each delivery facility type
```

Comment:
The placeholder values I've included here are actually the percentage of each facility type that have azithromycin, not the percentage of pregnancies that receive it. So we might need to do further data seeking if we want facility types to be more or less likely to actually give out the azithromycin to pregnant people.

Reply:
Could you add this information to the notes column so we know what the numbers actually represent and so we don't forget about possibly doing more data seeking?

Reply:
Also, grammar:
```suggestion
    - % of pregnancies that receive azithromycin in each delivery facility type
```